api/specs/storage/openapi.yaml

```diff
@@ -77,6 +77,13 @@ paths:
           minimum: 0
         name: user_id
         in: query
+      - required: false
+        schema:
+          title: Expand Dirs
+          type: boolean
+          default: true
```

Comment:
MINOR: i think it needs a small description. Expand dirs you means that you show inside of the zip all the content?